lib/validation/schema.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Litmus.Validation.Schema do
+  @moduledoc false
+
+  @doc """
+  Module for checking and routing to individual schema specific validation
```

Comment:
This should go in a moduledoc

Reply:
You might not even need this as public documentation, so maybe just remove and leave `@moduledoc false`?